src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -28,6 +31,11 @@ module.exports = class ArtworkCommercialView extends Backbone.View
 
     @artwork = new Artwork artwork
 
+
+    params = qs.parse(location.search.substring(1))
+    if params.inquire is 'true'
+      @inquire()
```

Comment:
just in case you may want to also check that the user is logged in. That would prevent a logged in user/edge cases from having `?inquiry=true` as a query param.

Reply:
What’s the cost of performing `User.instantiate()` ?

Reply:
There shouldn't really be any cost, essentially checking if a user is logged in is as simple as checking `sd.CURRENT_USER?`. The `CurrentUser` model and all that just wrap it.

I usually use `CurrentUser.orNull()` https://github.com/artsy/force/blob/d64b7416cc57b3cc7dd88484e1d6f098a174f538/src/lib/current_user.coffee#L282

Reply:
Nice one, will use that 👍